### PR TITLE
add table + create login list

### PR DESCRIPTION
**Pull request by maikl2999**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -11,30 +11,66 @@ document.onscroll = function () {
...
+        tr.innerHTML = '<td>' + user.login + '</td>' + '<td>' + user.id + '</td>' + '<td>' + user.url + '</td>';
+
+        table.appendChild(tr);
+        tr.firstElementChild.classList.add('user-login');
+    }
+
+    table.classList.add('table');
+    bottomGit.parentNode.appendChild(table);
+
+    table.onclick = function (e) {
```
Comment (trubenko) [inline script.js]: Use table.addEventListener instead table.onclick!

File: script.js [hunk 2]
```diff
@@ -11,30 +11,66 @@ document.onscroll = function () {
 };
 
 let bottomGit = document.getElementById('get-users');
-console.log(bottomGit);
 
-bottomGit.onmouseover = function (e) {
-  bottomGit.classList.add('bt-mouseover');
+bottomGit.onclick = function (e) {
```
Comment (trubenko) [inline script.js]: Use addEventListener instead onclick!

File: script.js (hunk 1 above)
Comment (maikl2999) [reply script.js]: ok bro :(

File: script.js (hunk 2 above)
Comment (maikl2999) [reply script.js]: yep

**Pull request closed without merging**